src/scalems/radical/raptor.py

```diff
@@ -226,33 +188,42 @@ def __init__(self, configuration: Configuration):
                 )
         super(ScaleMSMaster, self).__init__()
 
-    def result_cb(self, requests: typing.Sequence[Request]):
+    def result_cb(self, requests: typing.Sequence[rp.TaskDescription]):
+        """SCALE-MS specific handling of tasks completed by the collaborating Worker(s).
+
+        .. todo:: Clarify TaskDescription fields that are unique to raptor tasks, and the details of the
+            result tuple.
+        """
         for r in requests:
             logger.info('result_cb %s: %s [%s]' % (r.uid, r.state, r.result))
 
-    def request_cb(self, requests: RequestInputList) -> RequestInputList:
+    def request_cb(
+            self,
+            requests: typing.Sequence[rp.TaskDescription]
```

Comment:
requests: list of dictionaries.
A field of the dict is `description`. This member is a dictionary representation of the TaskDescription that was submitted with the `scheduler` field identifying this Master instance. It will contain `args` and `kwargs` fields associated with the dispatching function.